linux/mount.go

```diff
@@ -570,11 +570,16 @@ func MountDeviceWithFileSystem(devPath string, mountPoint string, options []stri
 	return mount, nil
 }
 
-func MountNFSShare(source string, targetPath string, options []string) error {
+func MountNFSShare(source string, targetPath string, options []string, nfsType string) error {
 	log.Tracef(">>>>> MountNFSShare called with source %s target %s", source, targetPath)
 	defer log.Tracef("<<<<< MountNFSShare")
 
-	args := []string{source, targetPath}
+	// default nfs version to 4.0
+	if nfsType == "" {
```

Comment:
nit: name this nfsVersion?

Reply:
Changed comment to reflect as nfsType instead. As its referred as filesystem type in general.

```For  most types all the mount program has to do is issue a simple mount(2) system call, and no detailed knowledge of the filesystem type
              is required.  For a few types however (like nfs, nfs4, cifs, smbfs, ncpfs) an ad hoc code is necessary.  The nfs, nfs4, cifs, smbfs, and
              ncpfs  filesystems  have a separate mount program.  In order to make it possible to treat all types in a uniform way, mount will execute
              the program /sbin/mount.type (if that exists) when called with type type.```

Reply:
cool